tools/src/publish-packages/tasks/updateAndroidProjects.ts

```diff
@@ -33,26 +33,14 @@ export const updateAndroidProjects = new Task<TaskArgs>(
 
       const relativeGradlePath = path.relative(EXPO_DIR, gradlePath);
 
-      logger.log(
-        '  ',
-        `Updating ${yellow('version')} and ${yellow('versionCode')} in ${magenta(
-          relativeGradlePath
-        )}`
-      );
+      logger.log('  ', `Updating ${yellow('version')} in ${magenta(relativeGradlePath)}`);
 
       await transformFileAsync(gradlePath, [
         {
           // update version and versionName in android/build.gradle
           find: /\b(version\s*=\s*|versionName\s+)(['"])(.*?)\2/g,
           replaceWith: `$1$2${state.releaseVersion}$2`,
         },
-        {
-          find: /\bversionCode\s+(\d+)\b/g,
-          replaceWith: (match, p1) => {
-            const versionCode = parseInt(p1, 10);
-            return `versionCode ${versionCode + 1}`;
-          },
-        },
```

Comment:
I guess we should remove it from the repo altogether then 😉 .

Reply:
I'll do 😉